xml/common_support-levels_i.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE variablelist
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<variablelist xml:lang="en"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <varlistentry>
+  <term>L1</term>
+  <listitem>
+   <para>
+    Problem determination, which means technical support designed to provide
+    compatibility information, usage support, ongoing maintenance, information
+    gathering and basic troubleshooting using available documentation.
+   </para>
```

Comment:
Stylistically the repeated "which means" sounds a bit clunky. What I've seen in the Support Guide that I share with you looked like: 
**L3-Standard**. CSP partner provides L1/L2 support to end customer and SUSE provides Standard (12x5) L3 support to CSP partner.

Reply:
Now I'm wondering whether we should describe the levels here at all. PAYG Customers of the public cloud get support from the CSP. BYOS customers get as much support as they've purchased. This is described here: https://www.suse.com/support/handbook/ but I can't figure out how to navigate from https://www.suse.com/support/ so perhaps it's not public yet. We could just direct them to https://www.suse.com/support/ and leave it at that.

Reply:
> PAYG Customers of the public cloud get support from the CSP

This is not going to be true for very much longer. IMHO we should not write the documentation in a way that makes it obsolete in short order.

Generally I think we should cross reference rather than re-document via cut and paste.

Reply:
https://www.suse.com/support/handbook/ is public, you can navigate through the menu _Support→Resources →SUSE Support User Guide_. So it's save to link, which I now do. I'm no longer explaining the levels but only mention them.